lib/auth/tun.go

```diff
@@ -474,11 +474,12 @@ func (s *AuthTunnel) passwordAuth(
 	log.Infof("[AUTH] login attempt: user %q type %q", conn.User(), ab.Type)
 
 	switch ab.Type {
-	// regular user is trying to get in using his password+OTP token:
+	// user is trying to get in using his password only
 	case AuthWebPassword:
```

Comment:
why change the meaning of the constant? this means that old clients will skp the check. I would do the opposite - introduce `WihoutOTP` so clients work as expected